components/fastly_backends/main.tf

```diff
@@ -152,34 +169,24 @@ resource "fastly_service_v1" "backends-qa" {
     }
   }
 
+  # Force SSL at the edge:
   request_setting {
     name      = "Force SSL"
     force_ssl = true
   }
 
-  response_object {
-    name              = "robots.txt deny"
-    content           = file("${path.module}/deny-robots.txt")
-    request_condition = "path-robots"
-  }
-
+  # Set an 'X-Origin-Name' header for debugging & for logging to Papertrail:
   snippet {
     name    = "Shared - Set X-Origin-Name Header"
     type    = "fetch"
     content = file("${path.module}/app_name.vcl")
   }
 
-  snippet {
-    name    = "Fix Geolocation With Shielding On"
-    type    = "recv"
-    content = file("${path.module}/recv_geolocation.vcl")
-  }
-
+  # Write requests to this Fastly property to the provided Papertrail drain:
   papertrail {
     name    = "backend"
     address = element(split(":", var.papertrail_destination), 0)
     port    = element(split(":", var.papertrail_destination), 1)
-    format  = var.papertrail_log_format
+    format  = "%t '%r' status=%>s app=%%{X-Application-Name}o cache=\"%%{X-Cache}o\" country=%%{X-Fastly-Country-Code}o ip=\"%a\" user-agent=\"%%{User-Agent}i\" service=%%{time.elapsed.msec}Vms"
```

Comment:
Since we're now re-using the same `fastly_backends` service across all environments, I don't think it's as useful to pass in the `papertrail_log_format` variable here like we did before.

Reply:
@DFurnes so this string is dynamically building what would have previously been saved as `var.papertrail_log_format`?

Reply:
Yep, I've just directly copied in the string value we used to send for this (as `var.papertrail_log_format`) from `dosomething/main.tf`, `dosomething-qa/main.tf`, and `dosomething-dev/main.tf`. I think it makes more sense to keep here now, since we can define it one place for all three environments.

Reply:
Here's, for example, where we used to grab this [for the production environment](https://github.com/DoSomething/infrastructure/blob/13f1ec7367067a89746f605e75704aaa95eb397d/dosomething/main.tf#L56).

Reply:
Makes sense to me, love having this long thing just in one place!